server/src/strategy.js

```diff
@@ -11,59 +11,63 @@ passport.use(new Strategy({
   callbackURL: process.env.CLIENT_CALLBACK_URL,
   scope: ['identify', 'guilds'],
 }, async (accessToken, refreshToken, profile, done) => {
-  let user = await User.findOne({ userId: profile.id });
-  if (!user) {
-    const userGuilds = profile.guilds.filter(async (g) => {
-      const guilds = await axios.get(`https://discord.com/api/v6/users/${process.env.CLIENT_ID}/guilds`, {
-        headers: {
-          Authorization: `BOT ${process.env.BOT_TOKEN}`,
-        },
+  try {
+    let user = await User.findOne({ userId: profile.id });
+    if (!user) {
+      const userGuilds = profile.guilds.filter(async (g) => {
+        const guilds = await axios.get(`https://discord.com/api/v6/users/${process.env.CLIENT_ID}/guilds`, {
+          headers: {
+            Authorization: `BOT ${process.env.BOT_TOKEN}`,
```

Comment:
change this to BOT to Bot for the authorization header, else it returns 401 unauthorized

also require dotenv.config() at the top so that way this environment variable does not return undefined